beacon-chain/db/iface/interface.go

```diff
@@ -74,10 +74,10 @@ type ReadOnlyDatabase interface {
 		ctx context.Context, proposals []*slashertypes.SignedBlockHeaderWrapper,
 	) ([]*eth.ProposerSlashing, error)
 	PruneAttestations(
-		ctx context.Context, currentEpoch types.Epoch, historySize uint64,
+		ctx context.Context, currentEpoch types.Epoch, historyLength uint64,
```

Comment:
Shouldn't history length be `types.Epoch` as well? It's denominated in epoch right?